src/cli.rs

```diff
@@ -74,6 +74,25 @@ pub struct Opts {
     #[arg(long, overrides_with = "no_ignore_vcs", hide = true, action = ArgAction::SetTrue)]
     ignore_vcs: (),
 
+    /// Do not require a git repository to respect gitignores.
+    #[arg(
+        long,
+        overrides_with = "require_git",
+        hide_short_help = true,
+        // same description as ripgrep's flag: ripgrep/crates/core/app.rs
+        long_help = "By default, fd will only respect global gitignore rules, .gitignore rules,\
+                     and local exclude rules if fd detects that you are searching inside a\
+                     git repository. This flag allows you to relax this restriction such that\
+                     fd will respect all git related ignore rules regardless of whether you're\
+                     searching in a git repository or not.\n\n\
+                     This flag can be disabled with --require-git."
+    )]
```

Comment:
This contains several missing whitespace errors due to how you placed those backslashes (there should be a space in front). Please check the rendered version using `cargo run -- --help` or `fd --help`:
```
      --no-require-git
          By default, fd will only respect global gitignore rules, .gitignore rules,and local
          exclude rules if fd detects that you are searching inside agit repository. This flag
          allows you to relax this restriction such thatfd will respect all git related ignore
          rules regardless of whether you'researching in a git repository or not.
          
          This flag can be disabled with --require-git.
```

Also, I would prefer if we could put that long help text in the doc comment, like for most other options.

Reply:
How's this look?